Podfile.lock

```diff
@@ -187,4 +187,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 72c7fef62790017248355f22fb8b085855f9d70d
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```

Comment:
Maybe update your cocoa pods before merging?

Reply:
Oh good point, I think Reade and I have been going back in forth for like 5 PRs now.